_sass/components/_hero.scss

```diff
@@ -51,3 +51,10 @@
   height: 720px;
 }
 
+.hero-galatour {
+  height: 480px;
+  background: url(https://res.cloudinary.com/zaigezaigu/image/upload/v1670294545/zgzg-io-website/2023-%E6%98%A5%E6%99%9A%E6%B8%B8%E5%9B%AD/%E9%A6%96%E9%A1%B5%E8%83%8C%E6%99%AF/%E9%A6%96%E9%A1%B5%E8%83%8C%E6%99%AF_fjuk1m.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
```

Comment:
如果用background-image: url(xxxxxx) 就可以不用复制底下那几行了。
https://stackoverflow.com/questions/39607999/css-difference-between-background-and-background-image

（其实我上面L2也应该用background-image instead of background，你可以把那个改了）